lib/node/notification.dart

```diff
@@ -262,11 +263,7 @@ Future<String> promptUser(String prompt,
 
   // Focus the element.
   Timer.run(() {
-    // There's a bug in Dart's JS interop where we can't see custom elements
-    // well. In order to work around it, we get the raw JS object, wrap it in a
-    // JsObject, pass that to a semantic wrapper around TextEditorElement, and
-    // then call `focused()`.
-    JsObject obj = new JsObject.fromBrowserObject(uncrackDart2js(editorElement));
+    JsObject obj = new JsObject.fromBrowserObject(editorElement);
```

Comment:
Correct, something changed in either js interop or atom, but there's no 'o' field on custom Elements.